src/daemon/XorgDisplayServer.cpp

```diff
@@ -233,6 +233,23 @@ namespace SDDM {
         // log message
         qDebug() << "Display server stopped.";
 
+        QString displayStopCommand = mainConfig.XDisplay.DisplayStopCommand.get();
+
+        // create display setup script process
+        QProcess *displayStopScript = new QProcess();
```

Comment:
On 02-04-15 08:55:15, davidedmundson wrote:
>    In [1]src/daemon/XorgDisplayServer.cpp:
> > @@ -233,6 +233,23 @@ namespace SDDM {
> >          // log message
> >          qDebug() << "Display server stopped.";
> >
> > +        QString displayStopCommand = mainConfig.XDisplay.DisplayStopCommand.g
> et();
> > +
> > +        // create display setup script process
> > +        QProcess *displayStopScript = new QProcess();
>
>    leaks.
>

Then we have two leaks, as XorgDisplayServer::setupDisplay() does the
same.

--
Alberto Milone